### PR TITLE
Replace usbmount with udiskie

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Udiskie (& udisks) is faster, more configurable, and doesn't need building from source

Note: Currently untested

### CODE REVIEW

**Comment by WillB97** [comment]
We'll need to check if the runusb user can access the mountpoint

**Comment by WillB97** [comment]
Permissions is broken and `MOUNTPOINT_DIR` in runusb will need updating so metadata is found
```
robot@robot:~ $ sudo ls -l /media/
total 4
drwxr-x---+ 3 root root 4096 Jul 26 21:55 root
robot@robot:~ $ sudo ls -l /media/root/
total 16
drwxr-xr-x 7 root root 16384 Jan  1  1970 F96A-B13B
```

Also you haven't enabled the systemd service.

File: parts/robot.sh [hunk 1]
```diff
@@ -14,6 +14,22 @@ apt-get install -y $PACKAGES_DIR/*.deb
 apt-get install -y udiskie
 cp /tmp/packer-files/udiskie/udiskie.yml /etc/
 cp /tmp/packer-files/udiskie/udiskie.service /lib/systemd/system/
+systemctl enable udiskie.service
+
+# Configure udisks2 to allow mounting as other users
+mkdir -p /etc/udisks2
+cat >/etc/udisks2/mount_options.conf <<EOF
```
Comment (RealOrangeOne) [inline parts/robot.sh]: Suggestion: Instead, what if we keep the file as its own file, and inject variables as needed with `envsubst`? Might make the files easier to maintain

Comment (WillB97) [reply parts/robot.sh]: The variables you need to populate in the file are so tightly tied to here that this is likely to be harder to maintain as you need to update two locations

**Review by sedders123 — APPROVED** [review]

**Pull request merged**